src/probabilistic.jl

```diff
@@ -544,3 +544,85 @@ $DOC_DISTRIBUTIONS
 SphericalScore
 "$SphericalScoreDoc"
 const spherical_score = SphericalScore()
+
+
+# ---------------------------------------------------------------------
+# Continuous Boyce Index
+struct _ContinuousBoyceIndex 
+    nbins::Integer
+    bin_overlap::AbstractFloat
+    min::Union{AbstractFloat, Nothing}
+    max::Union{AbstractFloat, Nothing}
+    cor::Function
+    function _ContinuousBoyceIndex(; nbins = 101, bin_overlap = 0.1, min = nothing, max = nothing, cor = StatsBase.corspearman)
+        new(nbins, bin_overlap, min, max, cor)
+    end
+end
+
+ContinuousBoyceIndex(; kw...) = _ContinuousBoyceIndex(; kw...) |> robust_measure |> fussy_measure
+
+function (m::_ContinuousBoyceIndex)(ŷ::UnivariateFiniteArray, y::NonMissingCatArrOrSub; warn=true)
```

Comment:
The type annotation for `yhat` is too strict. Either remove it altogether, or you could do `::AbstractArray{<:UnivariateFinite}`. For consider

```julia
julia> y = categorical(rand("ab", 10), ordered=true);

julia> d = CategoricalDistributions.Distributions.fit(UnivariateFinite, y);

julia> yhat = fill(d, 10);

julia> ContinuousBoyceIndex()(yhat, y)
ERROR: MethodError: no method matching (::StatisticalMeasures._ContinuousBoyceIndex)(::Vector{…}, ::CategoricalVector{…})
The object of type `StatisticalMeasures._ContinuousBoyceIndex` exists, but no method is defined for this combination of argument types when trying to treat it as a callable object.
```